demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -409,6 +409,11 @@ def are_valid_files(self) -> str:
         # We only check pack dependencies for -g flag
         if self.validate_dependencies:
             self.validate_pack_dependencies()
+        if id_set_validations:
```

Comment:
drop a line and add a comment about this section.

Reply:
done